habitat-baselines/habitat_baselines/common/obs_transformers.py

```diff
@@ -1202,8 +1202,15 @@ def get_active_obs_transforms(
     config: "DictConfig",
 ) -> List[ObservationTransformer]:
     active_obs_transforms = []
-    obs_trans_conf = config.habitat_baselines.rl.policy.obs_transforms
-    if hasattr(config.habitat_baselines.rl.policy, "obs_transforms"):
+
+    # We assume for now that the observation space is shared among agents
```

Comment:
We don't assume this in general right? Only when using observation transformations?